### PR TITLE
Add Working Agreement.

### PR DESCRIPTION
**Pull request by civrot**
Add Working Agreement.

At @danchapman request I am adding a 'Working Agreement' doc based on today's discussions.

### CODE REVIEW

File: working_agreement/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+- The individual on call needs to be available by phone during off hours.
+- The individual needs to be within a half hour of a machine with internet access so that they are able to diagnose any problems.
+
+
+## The Force
+
+- __The Force__ is a two person team responsible for fixing and cleaning Honey Badgers and bugs.
+- Assignments occur during the weekly Engineer Staff meeting.
+- There is one back-end and one front-end engineer.
+- Each member has a backup.
```
Comment (jmdeldin) [inline working_agreement/README.md]: Do we determine this ahead of time?

File: working_agreement/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+- On call responsibility rotates on a weekly basis. 
+- The rotation occurs on Monday mornings.
+- The individual on call needs to be available by phone during off hours.
+- The individual needs to be within a half hour of a machine with internet access so that they are able to diagnose any problems.
+
+
+## The Force
+
+- __The Force__ is a two person team responsible for fixing and cleaning Honey Badgers and bugs.
```
Comment (jmdeldin) [inline working_agreement/README.md]: Team Badger Force?

File: working_agreement/README.md (hunk 1 above)
Comment (civrot) [inline working_agreement/README.md]: Um..yes. Does that need to be spelled out?

**Comment by mcnelson** [comment]
There's some overlap with the Wiki. Fine with me if it lives here, but lets keep it in one place.

**Comment by civrot** [comment]
Where is it in the wiki?

**Comment by civrot** [comment]
The wiki should be app specific info. This repo should be team specific info.

**Comment by civrot** [comment]
Can we merge this?

**Comment by jmdeldin** [comment]
Do we want to clarify that despite the existence of the Badger Force, the whole team still needs to be cognizant of HBs?

**Comment by civrot** [comment]
That's like saying you have to come to work everyday.

**Comment by civrot** [comment]
@danchapman merge?

**Comment by civrot** [comment]
Thanks

**Pull request merged**